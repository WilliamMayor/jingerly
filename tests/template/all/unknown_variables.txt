There is no a variable unknown={{ unknown }}, so this shouldn't change