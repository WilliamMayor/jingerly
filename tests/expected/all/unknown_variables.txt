There is no variable unknown={{ unknown }}, so this shouldn't change